docs/angular/api-angular/schematics/ngrx.md

```diff
@@ -6,7 +6,20 @@ Add an ngrx config to a project
 
 ```bash
 ng generate ngrx ...
+```
+
+By default, Nx will search for `ngrx` in the default collection provisioned in `angular.json`.
+
+You can specify the collection explicitly as follows:
 
+```bash
+ng g @nrwl/angular:ngrx ...
+```
+
+Show what will be generated without writing to disk:
+
+```bash
+ng g null ... --dry-run
```

Comment:
There is still null here. This should be ng g ngrx or ng g @nrwl/angular:ngrx --dry-run